HackYourFuture-CPH/Guide-IT
# renamed container folder files acording to the naming guidelines

# Description
renamed container folder files acording to the naming guidelines"
Fixes #186 

# How to test?

Check visualy that the renamed files follow the guidelines, and run  "npm run dev", "npm run storybook" to make sure that everything runs as it should
# Checklist
